packages/main/src/Select.ts

```diff
@@ -419,6 +419,7 @@ class Select extends UI5Element implements IFormInputElement {
 		this._applySelection();
 
 		this.style.setProperty(getScopedVarName("--_ui5-input-icons-count"), `${this.iconsCount}`);
+		this.style.setProperty(getScopedVarName("--_ui5_list_item_title_size"), "var(--sapFontSize)");
```

Comment:
this is pattern only for css variables that have dynamic calculated values. this could be moved to Option.css